frontend/src/components/ResourcesDisplayCard.jsx

```diff
@@ -14,6 +18,16 @@ export default function ResourcesDisplayCard({ handleClose }) {
     };
   }, []);
 
+  const changeResource = (id) => {
+    setCurrentResourceId(id);
+  };
+
+  const currentResource = resources.find(
```

Comment:
nit: this can return `undefined`, don't see any guards for this in the following code? 

by guards i mean 

```jsx
currentResource ? <Content /> : <Fallback />
```